index.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
     getAccessorByteStride : require('./lib/getAccessorByteStride'),
     getBinaryGltf : require('./lib/getBinaryGltf'),
     getUniqueId : require('./lib/getUniqueId'),
+    gltfStatistics : require('./lib/gltfStatistics'),
```

Comment:
I think this should probably be named something else. For example when `gltfDefaults` was added to gltf-pipeline, it became `addDefaults`. Maybe something like `computeStatistics`.